website/projects/scimaps/src/app/pages/about/about.component.spec.ts

```diff
@@ -9,6 +9,8 @@ import { Profile } from '../../core/models/profile';
 export function getProfiles(numberOfProfiles: number): Profile[] {
   const profiles: Profile[] = [];
   for (let i = 0; i <= numberOfProfiles; i++) {
+    const roles = ['curator', 'advisory_board', 'ambassador'];
+    const index = Math.floor(Math.random() * 3);
```

Comment:
Not a good idea to use randomly generated roles for testing purposes

Reply:
Now created profiles with specific roles. 
Removed the randomly generated roles.